README.md

```diff
@@ -12,6 +13,16 @@ the Operating System images:
 - Qemu-kvm
 - virtio drivers (needed for Qemu to work with disks)
 
+## Host requirements
+
+### Minimal
+For most OS image builds, all you should really need is Docker on a reasonably modern Linux distro with KVM virtualization support.
+
+### ESXi special requirements
+To build ESXi images we depend on special bridged networking provided by libvirt-daemon.
```

Comment:
e.g. `sudo apt install bridge-utils libvirt-daemon-system libvirt-clients bridge-utils`

Reply:
I was nervous about how many different distros to explicitly document for.
I tested on NixOS so I didn't have the commands for e.g. Ubuntu handy.